ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf

```diff
@@ -0,0 +1,60 @@
+SectionDevice."Speaker" {
+        Comment "Speakers"
+        Value {
+               PlaybackChannels "8"
+               PlaybackPCM "hw:ALC1220VBDT,0"
+               JackControl "Speaker Jack"
+               PlaybackMixerElem "Speaker"
+        }
+}
+
+SectionDevice."Headphone" {
+        Comment "Front Headphone"
+        Value {
+                PlaybackChannels "2"
+                PlaybackPCM "hw:ALC1220VBDT,1"
+                JackControl "Front Headphone Jack"
+                PlaybackMixerElem "Front Headphone"
+        }
+}
+
+SectionDevice."SPDIF" {
+        Comment "S/PDIF Out"
+        Value {
+                PlaybackPriority 1
```

Comment:
Higher priority means preferred. Use 100 for Speaker, 200 for SPDIF, 300 for Headphones. The logic is that PA should pick the Headphones when connected.

Reply:
Done, but note that it seems that PA doesn't pick this up. I think it only resolves priority of multiple ports of the same sound card, but here each playback/capture stream appears as a different sound card.